src/core/Notification/InlineNotification/InlineNotification.test.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { InlineNotification } from './InlineNotification';
+import { axeTest } from '../../../utils/test';
+
+describe('props', () => {
+  describe('children', () => {
+    const notificationWithElementChild = (
+      <InlineNotification labelText="Lorem ipsum dolor sit">
+        <div>Test notification</div>
+      </InlineNotification>
+    );
+    it('has the given content', () => {
+      const { container } = render(notificationWithElementChild);
+      expect(container.firstChild).toHaveTextContent('Test notification');
+    });
+
+    it('should match snapshot', () => {
+      const { container } = render(notificationWithElementChild);
+      expect(container).toMatchSnapshot();
+    });
+  });
+  describe('labelText className Test', () => {
```

Comment:
This `describe` block should be named after the prop name like others, so it should be just `labelText` to be consistent with other props.

Reply:
renamed to be: 
describe('labelText', () => {